inst/rmarkdown/templates/thesis_pdf/resources/template.tex

```diff
@@ -37,6 +37,7 @@
 $endif$
 
 %%%%% FONTS
+\RequirePackage{droidserif} % Droid Serif font for body
```

Comment:
Shouldn't this follow the `{fontenc}` and `{microtype}` lines for best effects?

Reply:
According to [the TUG font catalogue](https://tug.org/FontCatalogue/droidserif/) one sould put it in front of `fontenc`. As for `microtype` multiple examples on the web put it after the font package. However, if I put the font package after both of the packages you mention it changes all font types to droid, which is not what we want.